rosidl_pycommon/rosidl_pycommon/__init__.py

```diff
@@ -201,3 +219,14 @@ def _expand_template(template_name, **kwargs):
               file=sys.stderr)
         raise
     interpreter.invoke('afterInclude')
+
+
+def _expand_type_hash(variable_name, hex_string, indent=0):
```

Comment:
I'm having trouble following this code - are you able to provide a docstring that would help review the code?

Reply:
Yup that's exactly right (the `{{` is the `empy` template way to output a literal `{`). I wanted to do the linebreaks just for readability in the generated code, I think it looks nicer. This helper function is so that I don't repeat myself in the few places it's needed in templates.

I'll add a docstring telling about params and output.